### PR TITLE
Scrum master and product owner in layman

### PR DESCRIPTION
**Pull request by marbeann**

### CODE REVIEW

File: 2_Operations/scrum.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Our work process is based on [Scrum](https://en.wikipedia.org/wiki/Scrum_(softwa
  * We work asynchronously, so keep scheduled meetings to the bare minimum.
  * We schedule our work into two week long *sprints*.
  * We manage our sprints with GitHub and ZenHub.
+ 
+The role of a product owner is to defend the users while the role of a scrum master is to defend the developers. This means that a symbiotic relationship has to exist in order to create a balanced product.
```
Comment (zupo) [inline 2_Operations/scrum.md]:
```suggestion
To put it in simple terms: **product owner defends the users, scrum master defends the team**. It is the balance of these two struggles that maximizes the value output.
```

Comment (zupo) [reply 2_Operations/scrum.md]: Too VC-speak? :)

Comment (dmurko) [reply 2_Operations/scrum.md]: rofl, definitely :D

Comment (zupo) [reply 2_Operations/scrum.md]: Open for suggestions then :D

Comment (dmurko) [reply 2_Operations/scrum.md]: "Ideally, this relationship brings the best results for users while not overworking the developers."
